pom.xml

```diff
@@ -213,11 +204,12 @@
 					</exclusion>
 				</exclusions>
 			</dependency>
+
 			<dependency>
-				<groupId>net.jpountz.lz4</groupId>
-				<artifactId>lz4</artifactId>
-				<version>${lz4.version}</version>
-				<scope>compile</scope>
+				<groupId>org.yaml</groupId>
+				<artifactId>snakeyaml</artifactId>
+				<version>${snakeyaml.version}</version>
+				<scope>runtime</scope>
```

Comment:
Where are runtime dependencies used? Is the assembly build with runtime libs in mind?